src/index.html

```diff
@@ -5,8 +5,416 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="header">
+        <a href="#" class="header__logo">
+          <img src="images/logo.png" alt="Moyo логотип">
+        </a>
+
+        <nav class="nav">
+
+          <ul class="nav__menu">
+            <li class="nav__list">
+              <a
+                href="#"
+                class="nav__link is-active"
+              >
+                Apple
+              </a>
+            </li>
+
+            <li class="nav__list">
+              <a href="#" class="nav__link nav__link--color-hover">
+                Samsung
+              </a>
+            </li>
+
+            <li class="nav__list">
+              <a href="#" class="nav__link nav__link--color-hover">
+                Смартфоны и телефоны
+              </a>
+            </li>
+
+            <li class="nav__list">
+              <a
+                href="#"
+                class="nav__link nav__link--color-hover"
+                data-qa="nav-hover"
+              >
+                Ноутбуки и компьютеры
+              </a>
+            </li>
+
+            <li class="nav__list">
+              <a href="#" class="nav__link">Гаджеты</a>
+            </li>
+
+            <li class="nav__list">
+              <a href="#" class="nav__link nav__link--color-hover">Планшеты</a>
+            </li>
+
+            <li class="nav__list">
+              <a href="#" class="nav__link nav__link--color-hover">Фото</a>
+            </li>
+
+            <li class="nav__list">
+              <a href="#" class="nav__link nav__link--color-hover">Видео</a>
+            </li>
+
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <div class="container">
+
+        <div class="card" data-qa="card">
+
+          <img
+            src="images/imac.jpeg"
+            class="card__photo"
+            alt="iMac 27"
+          >
+
+          <span class="card__title">
```

Comment:
delete these spaces everywhere
empty lines between sibling blocks of code improve readability, but not between a parent and a child